### PR TITLE
[rearchitecture] Assign entity correctly

### PR DESCRIPTION
**Pull request by jonathanmetzman**

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Bad this isn't caught by linting.

File: src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py [hunk 1]
```diff
@@ -1981,7 +1982,10 @@ def utask_preprocess(fuzzer_name, job_type, uworker_env):
   environment.set_value('PROJECT_NAME', data_handler.get_project_name(job_type),
                         uworker_env)
   targets_count = ndb.Key(data_types.FuzzTargetsCount, job_type).get()
-  fuzz_task_input = uworker_msg_pb2.FuzzTaskInput(targets_count=targets_count)
+  fuzz_task_input = uworker_msg_pb2.FuzzTaskInput()
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py:1985]: We get two footguns for the price of one in the original line!

**Comment by jonathanmetzman** [comment]
I think the PR this is fixing was particularly bad because it was authored before the proto refactor, so it was not written with the correct usage in mind.

File: src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py [hunk 2]
```diff
@@ -1981,7 +1982,10 @@ def utask_preprocess(fuzzer_name, job_type, uworker_env):
...
+  if targets_count:
```
Comment (letitz) [inline src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py:1986]: Is this accounting for the case where `targets_count` can be `None`?

Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py:1986]: I didn't. PTAL again. I'm now accounting for this.

**Pull request merged**